packages/number-field/stories/number-field.stories.ts

```diff
@@ -13,10 +13,11 @@ governing permissions and limitations under the License.
 import { html, TemplateResult } from '@spectrum-web-components/base';
 import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
 
-import '@spectrum-web-components/number-field/sp-number-field.js';
 import '@spectrum-web-components/field-label/sp-field-label.js';
-import { spreadProps } from '../../../test/lit-helpers.js';
+import '@spectrum-web-components/number-field/sp-number-field.js';
+import '@spectrum-web-components/slider/sp-slider.js';
```

Comment:
Question so I understand- what's the purpose of importing slider here, in the number field stories? I don't think the slider component is used within any of the number field stories, and it looks like there's a decorator that adds `sp-number-field` in the slider stories file. Is this import necessary to "connect" the two components together?

If it's easier to chat about this outside of the PR, that's fine too!

Reply:
oh good catch, forgot to remove, i had popped a slider in for easier testing in a single story